plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -135,6 +136,8 @@ func (c *PodSecurityPolicyPlugin) Admit(a admission.Attributes) error {
 			pod.ObjectMeta.Annotations = map[string]string{}
 		}
 		pod.ObjectMeta.Annotations[psputil.ValidatedPSPAnnotation] = pspName
+
+		a.SetAnnotations(AnnotationPluginName, "admit-policy", pspName)
```

Comment:
wondering whether we really have to make the PSP plugin aware of its own name. The admission chain loop could add this logic of using the right qualified key.

Reply:
Done.